src/shared/components/Partner/styles.ts

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+export const PartnerWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 140px;
+  background: #ffffff;
+  &:hover {
+    box-shadow: 0px 20px 30px rgba(233, 229, 221, 0.6);
+  }
+`;
+export const ImageElement = styled.img`
+  display: block;
+  width: 140px;
+  height: auto;
+
+  @media (max-width: 576px) {
+    width: 88.45px;
```

Comment:
we should not set strict width here - make it flexible for any amount of items with `flexbox`.

Reply:
I think you can use max-width: 88px; width: 100%;

Reply:
Fixed. I have taken padding from design. And made width for <img /> 100%.

![image](https://user-images.githubusercontent.com/5127736/79317639-9a1bbc00-7f0e-11ea-91d1-51b45aee20a9.png)
![image](https://user-images.githubusercontent.com/5127736/79317673-aa339b80-7f0e-11ea-8ddc-a902a45ddfe8.png)